float64 easting
float64 northing
float64 accuracy
uint64 gps_time
int32 gps_seqnum
uint8 gps_fix
